### ISSUE TITLE
colors api suggestion

### ISSUE THREAD

**Issue by rymohr** [gka/chroma.js#22]
Great library and loving the updates.  Had a quick suggestion though.

Since we already have a quick way to set the mode of a scale, how about providing a shorthand for grabbing colors like:

```
chroma.scale('RdYlGn').colors(5)
chroma.scale('RdYlGn').mode('hex').colors(5)
```

Instead of the current approach:

```
chroma.scale('RdYlGn').domain([0,1], 5).colors()
```

**Comment by gka** [comment]
implemented in 1.0.1!

**Comment by rymohr** [comment]
@gka :thumbsup: way to pull this one out of the dustbin.  This is still my favorite color library.
